uberfire-workbench/uberfire-workbench-client/src/main/java/org/uberfire/client/mvp/PlaceManagerImpl.java

```diff
@@ -278,6 +286,7 @@ private void goTo(final PlaceRequest place,
                          new Command() {
                              @Override
                              public void execute() {
+                                 //TODO eder, probably a bug place should be resolved.getPlace (conditional place)
```

Comment:
imho this should be resloved or have JIRA, but we should look into it.

Reply:
This TODO is not valid anymore, I'll delete it.